massa-models/src/hash.rs

```diff
@@ -1,41 +1,48 @@
-// Copyright (c) 2021 MASSA LABS <info@massa.net>
```

Comment:
we need to keep this for now

Reply:
also hash.rs is a confusing name. It's not a file dealing with hashes

Reply:
maybe prehashed.rs or something

Reply:
Fixed :heavy_check_mark: